src/test/regress/expected/multi_partitioning.out

```diff
@@ -4257,6 +4266,13 @@ ERROR:  non_partitioned_table is not partitioned
 CONTEXT:  PL/pgSQL function drop_old_time_partitions(regclass,timestamp with time zone) line XX at RAISE
 DROP TABLE non_partitioned_table;
 -- https://github.com/citusdata/citus/issues/4962
+SELECT stop_metadata_sync_to_node('localhost', :worker_1_port);
```

Comment:
Ops, we shouldn't need to stop metadata syncing for these tests. Is it failing if we don't stop?

Reply:
But it is only for one of the workers. So this way we test for both synced and non-synced case.
Is that not OK?

Reply:
Ah, missed that.

It is very good, but would be great to have a single line comment on why we stop